### ISSUE TITLE
Getting Started guide is not working

### ISSUE THREAD

**Issue by philschmid** [cargo-lambda/cargo-lambda#397]
Hello, 

I wanted to give `cargo lambda` a try and went through the [Getting Started](https://www.cargo-lambda.info/guide/getting-started.html) but it fails. 
I created my function with 
` cargo lambda new rust-lambda-multi-function-api --http`
and then run the watch command 
`cargo lambda watch`

and tried to send a request 
```bash
curl --request GET \
  --url http://localhost:9000/lambda-url/rust-lambda-multi-function-api
```

and returns a 500 
```json
{
	"type": "https://httpstatuses.com/500",
	"status": 500,
	"title": "Internal Server Error",
	"detail": "missing field `statusCode` at line 1 column 120"
}
```

with the following error in the logs

```
ERROR Lambda runtime invoke{requestId="d01e5eb8-bff4-4239-936f-e50f4c4a0f41" xrayTraceId="Root=1-64078e34-26d4f0fcb595febad8b14810;Parent=c2814342d26ff464;Sampled=1"}: Error("data did not match any variant of untagged enum LambdaRequest", line: 0, column: 0)
```

I hope this helps to fix the Getting Started. Its a shame if the first experience fails for a new user. 

FYI i also tried upgrading `lambda_http` and `lambda_runtime` to `0.7.3` without success

**Comment by greenwoodcm** [comment]
I just repro'ed.  I noticed that it does work with the following URL:
```
http://localhost:9000/lambda-url/rust-lambda-multi-function-api/
```
but does not work with the following:
```
http://localhost:9000/lambda-url/rust-lambda-multi-function-api
```
and it works with any path added to that base.

**Comment by philschmid** [comment]
Let me try again

**Comment by philschmid** [comment]
No success with our without the trailing `/`
![Bildschirmfoto 2023-03-08 um 09 17 58](https://user-images.githubusercontent.com/32632186/223659330-ca788728-c4ee-40d2-8fdd-cfb6fc11442c.png)
![Bildschirmfoto 2023-03-08 um 09 17 52](https://user-images.githubusercontent.com/32632186/223659333-5a71a858-240b-4cb1-b3da-39c533da78d6.png)

**Comment by philschmid** [comment]
Yes, 

```toml
[package]
name = "rust-lambda-multi-function-api"
version = "0.1.0"
edition = "2021"
```

Aren't the repro steps shared in https://github.com/cargo-lambda/cargo-lambda/issues/397#issue-1614058014 enough?

**Comment by philschmid** [comment]
Hay you tried a rest client other than curl?

**Comment by philschmid** [comment]
Thank you @calavera managed to get it to work too by removing `, default-features = false, features = [ "apigw_rest"]` from `lambda_http` which was added when using `cargo lambda new` and selecting `Amazon API Gateway REST API. 
<img width="735" alt="image" src="https://user-images.githubusercontent.com/32632186/224129481-ed9a18e5-3a62-4626-bada-8e7d451bd052.png">

Should this work with `cargo lambda watch`?
